src/table/tableview/table-view.component.js

```diff
@@ -3,6 +3,7 @@ angular.module('patternfly.table').component('pfTableView', {
     config: '<?',
     dtOptions: '<?',
     colummns: '<',
+    colvis: '<',
```

Comment:
> (2) However, current code use the hybrid mode to render datatables -- pre-populated html dom table drived by angular + core api of jquery datatables. Such an approach directly causes the lightweight solution as shown below not working.

Why doesn't this work?  dtOptions are passed directly to angular-datatables, unchanged by our code.  If angular-datatables supports dtOptions as you have outlined, then it should work.